content/en/blog/_posts/2022-04-12-prevent-unauthorised-volume-mode-conversion.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: blog
+title: 'Prevent unauthorised volume mode conversion'
+date: 2022-04-12
+slug: prevent-unauthorised-volume-mode-conversion-alpha
+---
+
+**Author:** Raunak Shah (Mirantis)
+
+Kubernetes v1.24 introduces a new alpha-level feature that prevents unauthorised users from modifying the volume mode of a [`PeristentVolumeClaim`](/docs/concepts/storage/persistent-volumes.md). This feature requires [`VolumeSnapshot`](/docs/concepts/storage/volume-snapshots.md) APIs with version `v6.0.0` onwards and `external-provisioner` version `v3.2.0` onwards.  
+
+### The problem
+
+As of Kubernetes 1.23, users can leverage the `VolumeSnapshot` feature, which GA'd in Kubernetes 1.20, to create a `PersistentVolumeClaim` (or `PVC`) from a previously taken `VolumeSnapshot`. This is done by pointing the `Spec.dataSource` parameter of the `PVC` to an existing `VolumeSnapshot` instance. 
```

Comment:
```suggestion
If you're using Kubernetes 1.23 or later, you can leverage the `VolumeSnapshot` API (stable since Kubernetes 1.20)
to create a new `PersistentVolumeClaim` (shortened as PVC) from a `VolumeSnapshot` that you had previously made.
You do this by defining the `spec.dataSource` parameter of the new PVC to refer to an existing
`VolumeSnapshot`. 
```

I wouldn't use code style for API kinds (PersistentVolumeClaim, etc). We usually don't in our docs.